manage.py

```diff
@@ -0,0 +1,98 @@
+# Management script for dwyl smart home
```

Comment:
@th0mas totally get that this is "WIP", but it's looking good so far and thanks for your _conscientiousness_! 🙌